app/src/main/java/com/example/stretchy/features/traininglist/ui/composable/TrainingListComposable.kt

```diff
@@ -319,6 +385,39 @@ private fun TrainingComposable(
     )
 }
 
+private fun getDataFromResult(
+    result: androidx.activity.result.ActivityResult,
+    context: Context,
+): String? {
+    val data: Uri? = result.data?.data
+    if (data != null && getFileName(
+            data,
+            context
+        ).contains(DataExporterImpl.dataTransportFileExt)
+    ) {
+        val inputStream = context.contentResolver.openInputStream(data)
+        val bufferedReader = BufferedReader(InputStreamReader(inputStream))
+        val stringBuilder = StringBuilder()
+        bufferedReader.forEachLine { line ->
+            stringBuilder.append(line)
+        }
+        val fileData = stringBuilder.toString()
```

Comment:
value declaration just to use it in return, simply
`return stringBuilder.toString()`